src/lib/components/identity/identity-details.svelte

```diff
@@ -45,6 +55,26 @@
                 <div class="fs-4 fw-bold">{identity?.username}</div>
                 <div class="text-secondary fw-bolder mt-1 text-break">{identity?.id}</div>
             </div>
+            <div class="d-flex flex-row justify-content-end role-padding">
+                <div class="pb-5">
+                    {#if userRole === UserRoles.Admin}
+                    <Dropdown size="sm">
+                        <DropdownToggle {caret}>
```

Comment:
I think this could simply just be (but please test before):
```suggestion
                        <DropdownToggle caret={!loading}>
```